src/Nethermind/Nethermind.Store/trieCases.txt

```diff
@@ -0,0 +1,48 @@
+For any cases where you need to test various trie layouts these are potential cases that you should test:
+
+Note 1) Nodes that are less than 32 bytes are not referenced by has but included in the referring node.
```

Comment:
wording around 'has', something is not grammatically correct.

Reply:
by hash